src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -16,32 +16,35 @@ redirects:
   - /docs/synthetics/synthetic-monitoring/getting-started
 ---
 
-Synthetic monitoring is a suite of automated, scriptable tools to monitor your websites, critical business transactions, and API endpoints. You can simulate user traffic to proactively detect and resolve outages and poor performance of critical endpoints before your customers notice.
+Our synthetic monitors act like crash test dummies for your websites, applications, and API endpoints. When you deploy a synthetic monitor, an API check or virtual browser instance runs through your most important workflows, testing critical endpoints so you can guarantee you're delivering the best customer experience. You don't even need to install an agent. 
 
-## Why it matters [#why]
+If you're interested, you can learn more about synthetic monitor checks and data ingest in our [Including synthetic monitor checks](/docs/synthetics/synthetic-monitoring/using-monitors/monitor-limits/) doc. 
 
-With synthetic monitoring, you can:
+## Synthetic monitors for proactive monitoring [#proactive]
 
-* Get the context of failures by connecting the availability and performance of endpoints to the underlying applications and infrastructure.
-* Easily diagnose if an issue stems from the network or AWS location, a slow third party resource, or the health of backend services or infrastructure.
-* Add synthetic monitoring into build automation and CI/CD pipelines to automatically track performance and check functionality for each deployment.
-* Expand your monitoring further with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
-* Monitor your API endpoints with [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests).
+Our synthetic monitors are more than just ping monitors. Synthetics can perform scriptable checks on your customer-facing or internal apps, down to the HTTP headers. The goal is to alert you to operational inefficiencies before anyone has a chance to notice.
 
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+With synthetic monitoring you can: 
+
+* Expand your browser monitoring with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
+* Run an [API test](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests) to check certificates, or to send `GET`, `POST`, or `HEAD` requests. 
```

Comment:
We should support any HTTP request type. I worry if we call out GET, POST, and HEAD that customers will think that it is all that is supported. 

We could also call out that additional node modules (https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/manage-monitor-runtimes/#Node16 and https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules/#NewRuntime) allow you to expand use case coverage of scripted API monitors.

Reply:
@bpeckNR Can you elaborate on what "expand use case coverage" means? It sounds like the new node modules (linked above) expand certain capabilities of the scripted API monitors? Can you be a bit more specific what features were ~added that should be called out?